app/screens/AppNavigator/screens/Transactions/TransactionsScreen.tsx

```diff
@@ -45,30 +42,39 @@ export function TransactionsScreen (): JSX.Element {
     })
   }
 
+  const onLoadMore = (): void => {
+    loadData(nextToken)
+  }
+
   useEffect(() => loadData(), [])
 
   return (
-    <FlatList
-      testID='transactions_screen_list'
-      style={tailwind('w-full')}
-      data={activities}
-      renderItem={TransactionRow(navigation)}
-      keyExtractor={(item) => item.id}
-      ListFooterComponent={hasNext ? LoadMore(loadData) : undefined}
-      refreshControl={
-        <RefreshControl
-          refreshing={loadingStatus === 'loading'}
-          onRefresh={loadData}
-        />
-      }
-    />
+    <>
+      {(activities.length === 0) && <EmptyTransaction navigation={navigation} handleRefresh={loadData} loadingStatus={loadingStatus} />}
```

Comment:
Use a ternary statement here

Reply:
```
    <>
      {
        (activities.length === 0)
          ? <EmptyTransaction navigation={navigation} handleRefresh={loadData} loadingStatus={loadingStatus} />
          : <FlatList
            testID='transactions_screen_list'
            style={tailwind('w-full')}
            data={activities}
            renderItem={TransactionRow(navigation)}
            keyExtractor={(item) => item.id}
            ListFooterComponent={hasNext ? LoadMore(onLoadMore) : undefined}
            refreshControl={
            <RefreshControl
              refreshing={loadingStatus === 'loading'}
              onRefresh={loadData}
            />
            }
            />
      }
    </>
```

Reply:
You can remove the React fragment <></> since it's only a single element. Then adjust everything to the left

Reply:
Removing the fragment will causes build error, can I change it back to if-else?

Reply:
Removed react fragment and parenthesis, change all attributes to single line passes eslint now